pkg/kubelet/volumes.go

```diff
@@ -142,6 +125,22 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.Pod) (kubecontainer.VolumeMap,
 		if builder == nil {
 			return nil, errUnsupportedVolumeType
 		}
+
```

Comment:
Minor nit: why not move this block to after `internal` is declared, so builder declaration is near the call to setup.